py/server/tests/test_numba_guvectorize.py

```diff
@@ -108,6 +109,19 @@ def g(x, res):
             t = empty_table(10).update(["X=i%3", "Y=(double)ii"]).group_by("X").update("Z=g(Y)")
         self.assertIn("g: Expected argument (1)", str(cm.exception))
 
+    def test_boxed_type_arg(self):
+        @guvectorize([(int64[:], int64, int64[:])], "(m),()->(m)", nopython=True)
+        def g(x, y, res):
+            for i in range(len(x)):
+                res[i] = x[i] + y
+
+        # make sure we don't get a warning about numpy scalar used in annotation
+        warnings.filterwarnings("error", category=UserWarning)
```

Comment:
How is this making sure that we don't get a warning?  I don't see an assert.

Reply:
If there is a warning we would get an Error that would fail the test.